src/display/general/init.go

```diff
@@ -113,69 +179,48 @@ func (page *MainPage) InitGeneral(numCores int) {
 			x / 6,
 		}
 	}
+}
 
-	// Initialize Plot for Network Chart
+func (page *MainPage) networkChartWidget() {
 	page.NetworkChart.Title = " Network data "
 	page.NetworkChart.TitleStyle = ui.NewStyle(ui.ColorClear)
-	page.NetworkChart.HorizontalScale = 1
-	page.NetworkChart.LineColors["RX"] = ui.ColorRed
-	page.NetworkChart.LineColors["TX"] = ui.ColorGreen
 	page.NetworkChart.BorderStyle.Fg = ui.ColorCyan
-	page.NetworkChart.Data["RX"] = []float64{0}
-	page.NetworkChart.Data["TX"] = []float64{0}
+	page.NetworkChart.Sparklines[0].TitleStyle.Fg = ui.ColorRed
+	page.NetworkChart.Sparklines[0].LineColor = ui.ColorRed
+	page.NetworkChart.Sparklines[1].TitleStyle.Fg = ui.ColorGreen
+	page.NetworkChart.Sparklines[1].LineColor = ui.ColorGreen
+}
 
-	if numCores > 8 {
-		page.CPUTable.Title = " CPU Usage "
-		page.CPUTable.BorderStyle.Fg = ui.ColorCyan
-		page.CPUTable.TitleStyle.Fg = ui.ColorClear
-		page.CPUTable.ColResizer = func() {
-			x := page.CPUTable.Inner.Dx()
-			page.CPUTable.ColWidths = []int{
-				x / 2,
-				x / 2,
-			}
-		}
-		page.CPUTable.Header = []string{"CPU", "Usage"}
-		page.CPUTable.ShowCursor = true
-		page.CPUTable.CursorColor = ui.ColorCyan
-	} else {
-		// Initialize Gauges for each CPU Core usage
-		for i := 0; i < numCores; i++ {
-			tempGauge := widgets.NewGauge()
-			tempGauge.Title = " CPU " + strconv.Itoa(i) + " "
-			tempGauge.Percent = 0
-			tempGauge.BarColor = ui.ColorBlue
-			tempGauge.BorderStyle.Fg = ui.ColorCyan
-			tempGauge.TitleStyle.Fg = ui.ColorWhite
-			tempGauge.LabelStyle.Fg = ui.ColorWhite
-			page.CPUCharts = append(page.CPUCharts, tempGauge)
+func (page *MainPage) cpuGaugeWidget(numCores int) {
+	page.CPUGauge.Title = " CPU Gauge "
```

Comment:
Could this be renamed to something more appropriate like "Per CPU Usage"? I'm not particularly fond of "CPU Gauge"